optuna/_transform.py

```diff
@@ -38,6 +38,9 @@ class _SearchSpaceTransform:
             space by one step. This allows fair sampling for values close to the bounds.
             Should always be :obj:`True` if any parameters are going to be sampled from the
             transformed space.
+        transform_0_1:
+            If :obj:`True`, apply a linear transformation to the bounds and parameters so that
+            they are in the unit cube.
```

Comment:
Perhaps we should fix the behavior without introducing this option, if this problem is a bug. What do you think?

Reply:
I'm sorry. I misunderstood the change. Let me resolve this.